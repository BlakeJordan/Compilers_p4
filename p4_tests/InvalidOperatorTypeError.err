7,11: Arithmetic operator applied to incompatible operands
7,7: Invalid assignment operation
8,5: Bad return value
Type checking failed
