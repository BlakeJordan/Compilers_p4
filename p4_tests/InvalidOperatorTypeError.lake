int f() {
    bool a;
    bool b;
    bool c;
    a = true;
    b = false;
    c = a + b;
    return c;
}