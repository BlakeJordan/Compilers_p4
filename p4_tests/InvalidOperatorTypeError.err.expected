7,9: Arithmetic operator applied to invalid operand
7,13: Arithmetic operator applied to invalid operand
8,12: Bad return value
Type checking failed